testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -989,6 +990,51 @@ public void testView()
                 .doesNotContain(testView);
     }
 
+    @Test
+    public void testRefreshView()
+    {
+        if (!hasBehavior(SUPPORTS_REFRESH_VIEW)) {
+            if (hasBehavior(SUPPORTS_CREATE_VIEW)) {
+                try (TestView testView = new TestView(getQueryRunner()::execute, "test_view", " SELECT * FROM nation")) {
+                    assertQueryFails("ALTER VIEW %s REFRESH".formatted(testView.getName()), "This connector does not support refreshing view definition");
+                }
+            }
+            else {
+                assertQueryFails("CREATE VIEW sample_view AS SELECT * FROM nation", "This connector does not support creating views");
+            }
+            return;
+        }
+
+        if (!hasBehavior(SUPPORTS_CREATE_TABLE) && !hasBehavior(SUPPORTS_ADD_COLUMN)) {
+            throw new AssertionError("Cannot test ALTER VIEW REFRESH without CREATE TABLE, the test needs to be implemented in a connector-specific way");
+        }
+
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_table", "(id BIGINT, column_to_dropped BIGINT, column_with_comment BIGINT)", ImmutableList.of("1, 2, 3"));
+                TestView view = new TestView(getQueryRunner()::execute, "test_view", " SELECT * FROM %s".formatted(table.getName()))) {
+            assertQuery("SELECT * FROM %s".formatted(view.getName()), "VALUES (1, 2, 3)");
+
+            assertUpdate("COMMENT ON COLUMN %s.column_with_comment IS 'test comment'".formatted(view.getName()));
```

Comment:
Should we check `SUPPORTS_COMMENT_ON_TABLE` and `SUPPORTS_RENAME_COLUMN` behaviors?